sycl/doc/extensions/proposed/sycl_ext_intel_cache_controls.asciidoc

```diff
@@ -98,6 +98,13 @@ Below is a list of new compile-time constant properties supported with
 ```c++
 namespace sycl::ext::intel::experimental {
 
+enum class cache_level {
```

Comment:
```suggestion
enum class cache_level : /*unspecified*/ {
```

Reply:
By putting "unspecified" in the spec, each vendor has the freedom to choose whatever integral base type they want.  This is how the core SYCL specification is written.  None of the enumerations have a specified integral base type.

Reply:
Here we have vendor (intel) extension, and we can choose integral base type to be `uint8_t`. Right?

Reply:
Yes, our implementation can use any integral base type we want.  The spec should still say "unspecified", so that applications don't bake-in assumptions on a specific type.  This gives us the freedom to change the base type later if necessary.